logging/logging_test.go

```diff
@@ -473,38 +473,27 @@ func TestPing(t *testing.T) {
 	}
 }
 
-func TestLogsAndDelete(t *testing.T) {
-	t.Skip("https://github.com/googleapis/google-cloud-go/issues/1654")
-
-	// This function tests both the Logs and DeleteLog methods. We only try to
-	// delete those logs that we can observe and that were generated by this
-	// test. This may not include the logs generated from the current test run,
-	// because the logging service is only eventually consistent. It's
-	// therefore possible that on some runs, this test will do nothing.
+func TestDeleteLog(t *testing.T) {
 	ctx := context.Background()
-	it := aclient.Logs(ctx)
-	nDeleted := 0
-	for {
-		logID, err := it.Next()
-		if err == iterator.Done {
-			break
-		}
-		if err != nil {
-			t.Fatal(err)
-		}
-		if strings.HasPrefix(logID, testLogIDPrefix) {
-			if err := aclient.DeleteLog(ctx, logID); err != nil {
-				// Ignore NotFound. Sometimes, amazingly, DeleteLog cannot find
-				// a log that is returned by Logs.
-				if status.Code(err) != codes.NotFound {
-					t.Fatalf("deleting %q: %v", logID, err)
-				}
-			} else {
-				nDeleted++
-			}
+	initLogs()
+	c, a := newClients(ctx, testProjectID)
+	defer c.Close()
+	defer a.Close()
+	lg := c.Logger(testLogID)
+	err := lg.LogSync(ctx, logging.Entry{Payload: "hello"})
```

Comment:
nit: combine with if check below `if err := lg.LogSync(ctx, logging.Entry{Payload: "hello"}); err != nil { ...}`

Reply:
noted for future , thanks!